src/sentry/static/sentry/app/components/u2f/u2fContainer.tsx

```diff
@@ -47,7 +47,13 @@ class U2fContainer extends React.Component<Props, State> {
       <div className={className}>
         {authenticators.map(({id, ...other}) => {
           if (id === 'u2f' && other.challenge) {
-            return <U2fSign key={id} {...this.props} challengeData={other.challenge} />;
+            return (
+              <U2fSign
+                key={id}
+                {...this.props}
+                challengeData={other.challenge as U2fSign['props']['challengeData']}
```

Comment:
Not sure about this `as` here, could move the challenge data type into `app/types` so authenticator can use it, but then types would to import `u2f-api`. Thoughts?

Reply:
It should be fine if it's just importing the type, yeah?

Reply:
I would move the challenge data to types and Imo it's fine to import the `u2f-api `